src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/repository/entity/User.kt

```diff
@@ -27,15 +28,38 @@ class User(
   @Column(value = "password")
   private var password: String? = null
 
+  @Column(value = "enabled")
+  private var enabled: Boolean = false
+
+  @Column(value = "inactive_reason")
+  var inactiveReason: String? = null
+
+  @Column(value = "email")
+  var email: String? = null
+
+  @Column(value = "first_name")
+  var first_Name: String? = null
```

Comment:
We don't have a convention of using under scores in variable names?

Reply:
Changed to firstName